.github/workflows/release.yml

```diff
@@ -0,0 +1,118 @@
+name: 𝌚 Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      force-publish:
+        required: true
+        type: boolean
+        description: Publish Releases at Anytime
+
+  workflow_run:
+    workflows: [ 🧪 Tests and Checks ]
+    branches: [main]
+    types: [ completed ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main' && github.repository_owner == 'wnfs-wg'
+
+    steps:
+      - name: Run release-please
+        id: release
+        uses: google-github-actions/release-please-action@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          default-branch: main
+          command: manifest
+          extra-files: |
+            Cargo.toml
+
+  publish-release-crates:
+    if: ${{ needs.release.outputs.releases_created || github.event.inputs.force-publish }}
+
+    runs-on: ubuntu-latest
+    needs: [ release-please ]
+
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Cache Project
+        uses: Swatinem/rust-cache@v2
+
+      - name: Install Rust Toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          override: true
+          profile: minimal
+          toolchain: stable
+
+      - name: Verify Publishing of crate
+        uses: katyo/publish-crates@v1
+        with:
+          dry-run: true
+
+      - name: Cargo Publish to crates.io
+        uses: katyo/publish-crates@v1
+        with:
+          registry-token: ${{ secrets.CARGO_REGISTRY_TOKEN }}
```

Comment:
@walkah @appcypher @matheus23, we need to set this in secrets. For publishing, it's prob set to run w/ @matheus23's token, but we should solve this once and for all.

Reply:
I think it's only the `skip-ratchet` crate that I own. I don't think this is my token.

I see that there's a `crates-token` in our 1password, but which account is that associated with? There's a `fission-suite/Development` owner on the `wnfs` crate, but I can't add `fission-suite/Development` as an owner to `skip-ratchet`.

Anyway. What I wrote was mostly about `skip-ratchet`. For `wnfs` I think we should be able to just switch out the secret here and it should work, right? Since we already have `fission-suite/Development` as one of the owners of the `wnfs` crate.

Reply:
@matheus23 yeah, let's hope :). @walkah would know best.

Reply:
I have a token but it is a personal access token. We need the `fission-suite/Development` token and there is an org-level `CARGO_REGISTRY_TOKEN` secret set that I would expect to work.

Reply:
yeah, there are definitely not team tokens, but if it's from the owner, we're good.